applications/TrilinosApplication/custom_processes/trilinos_levelset_convection_process.h

```diff
@@ -103,11 +103,10 @@ class TrilinosLevelSetConvectionProcess
         KRATOS_TRY
 
         const int row_size_guess = (TDim == 2 ? 15 : 40);
-        auto p_builder_and_solver = Kratos::make_shared< TrilinosBlockBuilderAndSolver<TSparseSpace,TDenseSpace,TLinearSolver>>(
+        BaseType::mpBuilderAndSolver = Kratos::make_shared< TrilinosBlockBuilderAndSolver<TSparseSpace,TDenseSpace,TLinearSolver>>(
```

Comment:
I'd avoid saving this as a member variable as it is only used for the construction of the NR strategy. See below.